civihr_employee_portal/civihr_leave_absences/civihr_leave_absences.module

```diff
@@ -74,8 +74,11 @@ function civihr_leave_absences_init() {
  *    Custom markup from template file.
  */
 function civihr_leave_absences_get_markup($resource = '') {
-
-  civicrm_resources_load('org.civicrm.reqangular', ['reqangular.min.js']);
+  // Loading all the footer js files.
+  $scripts = drupal_get_js('footer');
```

Comment:
<img width="1278" alt="screen shot 2017-03-27 at 2 28 57 pm" src="https://cloud.githubusercontent.com/assets/2689257/24348558/be535d1e-12f9-11e7-8122-675a22269d92.png">
we are not getting that error.